gffquant/annotation/count_writer.py

```
@@ -107,69 +111,92 @@ def compile_block(raw, lnorm, scaling_factors):
+            #         raise TypeError(f"Weird key: {str(item)}")
+            #     (cid, fid), fcounts = item
+            #     if (report_unseen or fcounts.sum()) and cid == category_id:
+            #         CountWriter.write_row(feature_names[fid], fcounts, stream=feat_out,)
+
+            empty_row = np.zeros(6, dtype=CountMatrix.NUMPY_DTYPE)
+            for feature in features:
+                key = (category_id, feature.id)
+                if counts.has_record(key):
+                    row = counts[key]
+                else:
+                    row = empty_row
```

Ruff (0.8.2)

Use ternary operator `row = counts[key] if counts.has_record(key) else empty_row` instead of `if`-`else`-block

Replace `if`-`else`-block with `row = counts[key] if counts.has_record(key) else empty_row`

(SIM108)